File: app/src/main/java/com/example/jy/jyweather/entity/UpdateBean.java
package com.example.jy.jyweather.entity;

import com.google.gson.annotations.SerializedName;

/**
 * Update实体类
 * <p>
 * Created by Yang on 2017/12/11.
 */
public class UpdateBean {

    @SerializedName("loc")
    private String loc; //当地时间

    @SerializedName("utc")
    private String utc; //UTC时间

    public String getLoc() {
        return loc.split(" ")[1];
    }

    public String getUtc() {
        return utc;
    }
}

File: app/src/main/java/com/jy/weather/entity/UpdateBean.kt
package com.jy.weather.entity

import com.google.gson.annotations.SerializedName

/**
 * Update实体类
 *
 * Created by Yang on 2017/12/11.
 */
data class UpdateBean(
    @SerializedName("loc")
    private val loc: String,    //当地时间

    @SerializedName("utc")
    val utc: String             //UTC时间
) {
    fun getLoc(): String = loc.split(" ".toRegex()).getOrElse(1) { "unknown" }
}